openbb_terminal/stocks/fundamental_analysis/fmp_model.py

```diff
@@ -585,3 +585,58 @@ def clean_metrics_df(data: pd.DataFrame, num: int, mask: bool = True) -> pd.Data
     )
 
     return data
+
+
+@log_start_end(log=logger)
+@check_api_key(["API_KEY_FINANCIALMODELINGPREP"])
+def get_filings(
+    pages: int = 1,
+) -> pd.DataFrame:
+    """Get SEC Filings RSS feed, disseminated by FMP
+
+    Parameters
+    ----------
+    pages: range = 1
+        The range of most-rececnt pages to get entries from (1000 per page; maximum of 30 pages)
+
+    Returns
+    -------
+    pd.DataFrame
+        Dataframe of results
+    """
+    df: pd.DataFrame = ()
```

Comment:
Note the type hinting here doesn't make sense.  You say df is a dataframe but equal to an empty tuple.  You can just get rid of this definition here, as you definne df as the concat later,

Reply:
Done